### PR TITLE
pybind: volume_client handle purge of directory names encoded in utf-8

### PR DESCRIPTION
**Pull request by josecastroleon**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/ceph_volume_client.py [hunk 1]
```diff
@@ -731,7 +731,7 @@ def purge_volume(self, volume_path, data_isolated=False):
             return
 
         def rmtree(root_path):
-            log.debug("rmtree {0}".format(root_path))
+            log.debug(u"rmtree {0}".format(root_path))
```
Comment (tchaikov) [inline src/pybind/ceph_volume_client.py:734]: i'd suggest drop this change. as in python3, `"rmtree {0}"` is a unicode string. and we've dropped python2 support.

File: qa/tasks/cephfs/test_volume_client.py [hunk 2]
```diff
@@ -529,6 +529,9 @@ def test_purge(self):
         self.mount_a.run_shell(["touch", os.path.join(mount_path, "noperms")])
         self.mount_a.run_shell(["chmod", "0000", os.path.join(mount_path, "noperms")])
 
+        # A folder with non-ascii characters
+        self.mount_a.run_shell(["mkdir", os.path.join(mount_path, u"f\u00F6n")
```
Comment (tchaikov) [inline qa/tasks/cephfs/test_volume_client.py:533]:
```suggestion
        self.mount_a.run_shell(["mkdir", os.path.join(mount_path, "f\u00F6n")
```

File: src/pybind/ceph_volume_client.py (hunk 1 above)
Comment (tchaikov) [reply src/pybind/ceph_volume_client.py:734]: could you instead retarget this change to nautilus where we still support python2?

Comment (josecastroleon) [reply src/pybind/ceph_volume_client.py:734]: Moved to https://github.com/ceph/ceph/pull/36679, shall we close this one?

Comment (batrick) [reply src/pybind/ceph_volume_client.py:734]: Yes: https://github.com/ceph/ceph/pull/36679#issuecomment-694609332

**Pull request closed without merging**